frontend/public/components/storage/create-pvc.tsx

```diff
@@ -172,6 +183,11 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
 
     if (storageClass) {
       obj.spec.storageClassName = storageClass;
+
```

Comment:
Nit: extra line

Reply:
I have added it purposely in order to differentiate the two `ifs` and the comment